helm/examples/azure/log-analytics/loganalytics-hpcc-logaccessV2.yaml

```diff
@@ -0,0 +1,91 @@
+# Configures HPCC logAccess to target Azure Log Analytics Workspace
+global:
+  logAccess:
+    name: "Azure LogAnalytics LogAccess"
+    type: "AzureLogAnalyticsCurl"
+    #connection:
+      #All connection attributes are optional.
+      #It is preferable to provide connection values as secret values category 'esp', secret name 'azure_logaccess'
+      # NOTE: secret 'azure_logaccess' must include 'aad-client-secret' and it cannot be provided in configuration
+      #
+      #workspaceID: "XYZ"      #ID of the Azure LogAnalytics workspace to query logs from
+                               # Secret value equivalent: 'ala-workspace-id'
+      #clientID: "DEF"         #ID of Azure Active Directory registered application with api.loganalytics.io access - format: 00000000-0000-0000-0000-000000000000
+                               # Secret value equivalent: 'aad-client-id'
+      #tenantID: "ABC"         #The Azure Active Directory Tenant ID, required for KQL API access
+                               # Secret value equivalent: 'aad-tenant-id'
+    logMaps:
+    - type: "global"
+      storeName: "ContainerLogV2"
+      searchColumn: "LogMessage"
+      timeStampColumn: "hpcc_log_timestamp"
+      columnType: "dynamic"
+      columnMode: "ALL"
+    - type: "workunits"
+      searchColumn: "hpcc_log_jobid"
+      columnMode: "DEFAULT"
+      columnType: "string"
+    - type: "components"
+      storeName: "ContainerLogV2"
+      searchColumn: "ContainerName" # Container name happens to coincide with component name
```

Comment:
> This field ("components") means component type ?
> 
> Sometimes the container name happens to be something like the component type, but not specifically or universally, and we can't make it so, because there are multiple containers (with unique names) in a pod of the same type. Or, multiple processes within a container with different types.
> 
> I think we will need to map this to an actual logged component type, which we currently do not log, but we do have a jlog field type for: MSGFIELD_component.
> 
> Maybe this is beyond the scope of this PR, but we should open a JIRA and reference to revisit.

Right, we discussed this early in the process and acknowledged we need a 1-to-1 mapping, but this was a good enough placeholder until then. Created this jira to track: https://track.hpccsystems.com/browse/HPCC-30444

For this PR, the priority is to continue mapping the components log field type with the same information in a more performant fashion.